native-code/ios/index.md

```diff
@@ -8,9 +8,30 @@ crumb: iOS
 
 {% include toc-hide.html %}
 
+### Using Cocoapods
+
+The WebRTC framework  is published on [cocoapods.org](https://cocoapods.org/pods/GoogleWebRTC).
+The framework is built from tip-of-tree.
+To integrate it into your project add the following lines to your Podfile
```

Comment:
Maybe it would be good to add a paragraph about the framework not containing bitcode, how to disable building with bitcode in XCode and mention more clearly that if bitcode is needed one should build from source.

Reply:
Done